Gemfile

```diff
@@ -20,3 +22,11 @@ group :development, :test do
   gem 'better_errors'
   gem 'binding_of_caller'
 end
+
+group :test do
+  gem "faker", "~> 1.4.3"
+  gem "capybara", "~> 2.4.3"
+  gem "database_cleaner", "~> 1.3.0"
+  gem "launchy", "~> 2.4.2"
+  gem "selenium-webdriver", "~> 2.43.0"
```

Comment:
You may end up wanting `capybara-webkit` instead of selenium, but it'll be fine for now.